main.go

```diff
@@ -444,11 +445,10 @@ func matchFuzzy(create, destroy *strset.Set) (map[string]string, map[string]stri
 		for _, c := range candidates[1:] {
 			if bestCandidate.distance == c.distance {
 				if (bestCandidate.create == c.create) || (bestCandidate.destroy == c.destroy) {
+					errorMap[bestCandidate.create] = bestCandidate.destroy
+					errorMap[c.create] = c.destroy
 					return map[string]string{}, map[string]string{},
-						fmt.Errorf("ambiguous migration: {%s} -> {%s} or {%s} -> {%s}",
-							bestCandidate.create, bestCandidate.destroy,
-							c.create, c.destroy,
-						)
+						fmt.Errorf("ambiguous migration: %v", errorMap)
```

Comment:
Reason why this works is quite hidden: https://yourbasic.org/golang/sort-map-keys-values/
`Also, starting with Go 1.12, the fmt package prints maps in key-sorted order to ease testing.`